Rubberduck.Parsing/Grammar/VBAParser.g4

```diff
@@ -687,6 +698,7 @@ keyword :
      | ARRAY
      | ATTRIBUTE
      | BEGIN
+	 | BEGINPROPERTY
```

Comment:
tab/spaces?

Reply:
Oh, I hate that the grammar uses spaces. Will correct it.